lms/static/scripts/frontend_apps/components/dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ArrowLeftIcon,
+  CaretRightIcon,
+  Link,
+} from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { Link as RouterLink } from 'wouter-preact';
+
+type BreadcrumbLink = {
+  title: string;
+  href: string;
+};
+
+export type DashboardBreadcrumbsProps = {
+  links: BreadcrumbLink[];
+};
+
+function BreadcrumbLink({ title, href }: BreadcrumbLink) {
+  return (
+    <RouterLink href={href} asChild>
+      <Link underline="hover" variant="text-light" classes="truncate">
+        <ArrowLeftIcon className="inline-block md:hidden mr-1 align-sub" />
+        {title}
+      </Link>
+    </RouterLink>
+  );
+}
+
+export default function DashboardBreadcrumbs({
+  links,
+}: DashboardBreadcrumbsProps) {
+  return (
+    <div
+      className="flex flex-row gap-0.5 w-full font-semibold"
+      data-testid="breadcrumbs-container"
+    >
+      {links.map(({ title, href }, index) => {
+        const isLastLink = index === links.length - 1;
+        return (
+          <span
+            key={`${index}${href}`}
+            className={classnames('gap-0.5', {
+              // In mobile devices, show only the last link
+              'md:flex hidden': !isLastLink,
+              'flex max-w-full': isLastLink,
+              // Distribute max width for every link as evenly as possible.
+              // Setting explicit classes here is required for Tailwind to
+              // properly detect they are being used.
+              'md:max-w-[50%]': links.length === 2,
+              'md:max-w-[33.333333%]': links.length === 3,
+              'md:max-w-[25%]': links.length === 4,
```

Comment:
Would fractions work here (`md:max-w-1/3` etc)?

Reply:
That's the first thing I tried, and it took me a while to know why it was not working 😅 

Apparently fraction width utilities are defined only for `w-??` classes, but not for `max-w-??` ones. Not sure why.

Reply:
OK. For the `1/3` case this level of precision isn't needed. `33%` will be fine.